src/1_utils.js

```diff
@@ -441,26 +441,37 @@ utils.scrapeOpenGraphContent = function(property, content) {
 
 /**
  * Search for hosted deep link data on the page, as outlined here https://dev.branch.io/getting-started/hosted-deep-link-data/guide/#adding-metatags-to-your-site
+ * Also searches for applink tags, i.e. <meta property="al:ios:url" content="applinks://docs" />
  */
 utils.scrapeHostedDeepLinkData = function() {
 	var params = {};
 
 	var metas = document.getElementsByTagName('meta');
 
 	for (var i = 0; i < metas.length; i++) {
-		if (!metas[i].getAttribute('name') || !metas[i].getAttribute('content')) {
+		if (!metas[i].getAttribute('name') && !metas[i].getAttribute('property') || !metas[i].getAttribute('content')) {
 			continue;
 		}
 
-		var name = metas[i].getAttribute('name');
-		var split = name.split(":");
+		if (metas[i].getAttribute('name')) {
+			var name = metas[i].getAttribute('name');
+			var split = name.split(':');
 
-		if ((split.length === 3) && (split[0] === 'branch') && (split[1] === 'deeplink')) {
-			params[split[2]] = metas[i].getAttribute('content');
+			if ((split.length === 3) && (split[0] === 'branch') && (split[1] === 'deeplink')) {
+				params[split[2]] = metas[i].getAttribute('content');
+			}
+		}
+		else if (metas[i].getAttribute('property')) {
+			var property = metas[i].getAttribute('property');
+
+			if (property === 'al:ios:url') {
+				params['$ios_deeplink_path'] = metas[i].getAttribute('content');
```

Comment:
Double check, but I'm pretty sure App Links urls include the URI scheme. (e.g. my-scheme://host/pathname)

We'll wanna strip the scheme out before setting it to the $deeplink_path.